### DISCUSSION TITLE
What is the most successful method to automate circular rebalancing?

### DISCUSSION THREAD

**Discussion by BonzoDogDooDah** [lightningnetwork/lnd#8074] [Q&A]
What is the most successful method to automate circular rebalancing?

We have tried the Torq and the LNDG app but so far they both do not wotk for us so we are still performing manual circular rebalancing using RTL. Please can someone suggest a tutorial on how to use Torq and or LNDG or even another method?

Thank you. :)

Orange_Bonzo

**Comment by hieblmi** [comment]
Instead of rebalancing you could also consider using lightning Loop's auto-loop feature to automatically acquire inbound or outbound liquidity on selected channels. Checkout the autoloop feature: https://docs.lightning.engineering/lightning-network-tools/loop/autoloop. This is also integrated into Lightning Terminal UI which should ease the setup of autoloop features.

**Comment by BonzoDogDooDah** [comment]
Hi Slyghtning,

Thanks for the reply.

We will try that. Can I ask you if that is an inexpensive way to rebalance channels? As at the moment we are paying about 125 ppm per 100k sats to perform a  rebalance that actually works as  most times any higher amount of sats will not unless we choose a 1 or 2 % fee?

Thank you :)

Orange_Bonzo
-- 
 Sent with Tutanota; enjoy secure & ad-free emails: 
 https://tutanota.com


9 Oct 2023, 19:11 by ***@***.***:

**Reply by hieblmi** [reply]
Loop-ins(acquiring outbound liquidity) are a lot cheaper than Loop-outs(acquiring inbound liquidity), but in any case they are both way cheaper for larger amounts than 1-2% in fees. Here are two quotes for loop-in and loop-out for _0.1 BTC_ I just did:
```
loop in quote --amt 10000000
Send on-chain:                           10000000 sat
Receive off-chain:                        9997179 sat
Estimated total fee:                         2821 sat
```
That's a fee of ~0.02% to acquire _0.1 BTC_ outbound liquidity on a certain channel.

```
loop out quote --amt 10000000
Send off-chain:                          10000000 sat
Receive on-chain:                         9986606 sat
Estimated total fee:                        13394 sat
```
That's a fee of ~0.13% to acquire _0.1 BTC_ inbound liquidity on a certain channel.

**Comment by BonzoDogDooDah** [comment]
Hi Slyghtning,

Thank you for the quick response. We will try this soon and then let you know how it went.

Do you recommend doing a loop out at 10000000 sats (10 Million) or should us newbie's try a lower amount?

Thank you.

Orange_Bonzo
-- 
 Sent with Tutanota; enjoy secure & ad-free emails: 
 https://tutanota.com


9 Oct 2023, 19:32 by ***@***.***:

**Reply by hieblmi** [reply]
10 Million should be no problem at all, you could try up to 1.2 BTC at once. Let me know how it goes and I am happy to help in case you face any issues or need larger swap amounts.

**Comment by BonzoDogDooDah** [comment]
Hi Slyghtning,

Will it work with 1 or 2 million sats?

Thank you.

Orange_Bonzo
-- 
 Sent with Tutanota; enjoy secure & ad-free emails: 
 https://tutanota.com


9 Oct 2023, 19:50 by ***@***.***:

**Comment by hieblmi** [comment]
Yes it will. The minimum is 250k. You can check your loop terms with `loop terms`.

**Comment by BonzoDogDooDah** [comment]
Hi Sly,

I know this is off the topic but my node partner has asked me if upgrading our Raspberry Pi 4 to a computer with more power would help to stop our Umbrel node from crashing so frequently? I said it may be due to LND still being in beta? If you have any free time could you please let us know?

Thank you.

Orange_Bonzo
-- 
 Sent with Tutanota; enjoy secure & ad-free emails: 
 https://tutanota.com


9 Oct 2023, 19:56 by ***@***.***:
